pymeasure/instruments/mock.py

```diff
@@ -22,15 +22,17 @@
 # THE SOFTWARE.
 #
 
-import numpy
 import time
+import numpy as np
 
 from pymeasure.adapters import FakeAdapter
 from pymeasure.instruments import Instrument
 
 
 class Mock(Instrument):
```

Comment:
Can you rename this to `FakeInstrument` while you're in there? I think it reduces potential confusion with ` mock.Mock`, and is more consistent with `FakeAdapter`.

Reply:
Would you like the `Mock` class renamed to `FakeInstrument` or would you like `Mock` to inherit from the currently existing `FakeInstrument` class? I have assumed the latter for the latest commit.

Reply:
Oh, yeah, of course, that would collide otherwise, sorry about the confusion. 
I suggest the following:
* Inherit from `FakeInstrument` as you chose already.
* Give `Mock` a more expressive name. It's a bit hard at this point as this mixes I/O features between different instrument types, so it does not cleanly matcha single one, say, an oscilloscope. The best I could come up with is `MultiFake` or `SwissFake` (like a multitool or Swiss knife). I'm open to better suggestions, otherwise `SwissFake` is probably most appropriate, as its ridiculousness also stands out a little, which is good for a dummy instrument. :D 
* Rename mock.py to fakes.py and move the `FakeInstrument` implementation there. This improves the structure a bit by keeping dummy/testing classes a bit away from `Instrument`. AFAICT, this only necessitates a changed import in `test_instruments.py` (I know `FakeAdapter` still resides with `Adapter`, but there are no other dummy classes around, so we IMO haven't reached the point where that should be extracted.)